lib/get-category-path.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var path = require('path'),
+  utility = require('./utility');
+
+
+/**
+ * Determines the category directory structure from the pattern's data
+ *
+ * @param {Object} patternObject  individual pattern's data (from /pattern-name/pattern.yml)
+ * @param {Object} options  pattern-importer options
+ *
+ * @return {String} categoryPath  internal path url to category
+ *
+ * @requires ./lib/utility
+ * @requires ./lib/category-name-converter
+ */
+exports.getCategoryPath = function (patternObject, options) {
```

Comment:
This code makes a determination of:

1. figures out the category and subcategory from the pattern's data file
2. figures out if those categories need to be converted to match the system where the pattern's are being imported into (if there is a different required directory naming structure in the destination environment like there is in Pattern Lab)
    
    A. we do not have intelligent control over the way Pattern Lab (a third-party tool) requires for it's category folder names and had to figure out a way to inject our patterns in there in the way that PLab requires

all that being said...@ericduran I'm very open to alternative ways to do this